app/models/ncr/work_order.rb

```diff
@@ -193,19 +193,35 @@ def system_approver_emails
     end
 
     def self.ba61_tier1_budget_mailbox
-      ENV['NCR_BA61_TIER1_BUDGET_MAILBOX'] || 'communicart.budget.approver@gmail.com'
+      users = User.with_role('BA61_tier1_budget_approver').where(client_slug: 'ncr')
+      if users.length == 0
+        raise "Missing User with role BA61_tier1_budget_approver -- did you run rake db:migrate and rake db:seed?"
+      end
+      users.first.email_address
```

Comment:
Since all of these methods do basically the same thing, can we share the logic somewhere?